subsys/bluetooth/audio/vocs_internal.h

```diff
@@ -0,0 +1,85 @@
+/** @file
+ *  @brief Internal APIs for Bluetooth VOCS.
+ *
+ * Copyright (c) 2020 Bose Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_VOCS_INTERNAL_
+#define ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_VOCS_INTERNAL_
+#include <zephyr/types.h>
+
+#if defined(CONFIG_BT_VOCS)
+#define VOCS_MAX_DESC_SIZE CONFIG_BT_VOCS_MAX_OUTPUT_DESCRIPTION_SIZE
```

Comment:
This header file is using both `bt_vocs` and `vocs` as the prefix, they should probably be all `bt_vocs`.